modules/tinymce/src/core/test/ts/browser/delete/InlineFormatDeleteTest.ts

```diff
@@ -904,7 +896,8 @@ describe('browser.tinymce.core.delete.InlineFormatDelete', () => {
           });
         })
       );
-      TinyAssertions.assertCursor(editor, [ 0, 0, 0, 0 ], 0);
+      const selPath = browser.isFirefox() ? [ 0, 0 ] : [ 0, 0, 0, 0 ];
```

Comment:
well at least the formats match now, even if the cursor position doesn't.

Reply:
Yeah, the cursor positions matching before was purely a coincidence. The difference in cursor positions is because of the difference in structures, which is due to Firefox not needing any custom caret formats being inserted as it natively preserves the original formats, unlike Chrome or Safari.